ddtrace/profiling/exporter/pprof.pyx

```diff
@@ -369,11 +384,10 @@ class PprofExporter(exporter.Exporter):
     ):
         # type: (...) -> _stack_event_group_key_T
 
-        if event.trace_type == ext.SpanTypes.WEB.value:
-            trace_resource = self._none_to_str(event.trace_resource)
-        else:
-            # Do not export trace_resource for privacy concerns.
-            trace_resource = ""
+        trace_resource = None
+        # Do not export trace_resource for non Web spans for privacy concerns.
```

Comment:
we lose this comment on the duplicate code below.

maybe introducing a small helper to this file might help DRY it up a bit?

```python
def _get_event_trace_resource(event):
    trace_resource = None
    # Do not export trace_resource for non Web spans for privacy concerns.
    if event.trace_resource_container and event.trace_type == ext.SpanTypes.WEB.value:
        (trace_resource,) = event.trace_resource_container
    return trace_resource


... ,
self._none_to_str(_get_event_trace_resource(event)),
... ,
```


I don't see this as a blocker, just an easy win to clean this up a little bit.

Reply:
fixed